docs/client-specifications.rst

```diff
@@ -87,7 +87,7 @@ Returns the following response for the collection ``{cid}`` in the bucket ``{bid
 
 .. note::
 
-    The ``_expected={}`` querystring parameter is mandatory but can be set to ``0`` if unknown. See section below about cache busting.
+    The ``_expected={}`` querystring parameter is mandatory. Either you pass the current collection timestamp value obtained when polling for changes in order to bust the CDN cache, or you use a hard-coded value (eg. ``0``) and rely on the cache TTL. See section below about cache busting.
```

Comment:
I'm not quite sure what "current collection timestamp" means.  Is it the timestamp of the last fetch?  The timestamp returned in the data for the last fetch?  The current time?  Whichever one it is, please spell out how it would work to bust the cache.

Can you lay out a scenario when you would want to bust the cache while polling?  In general, it seems like I wouldn't want to do that.

Also, how can we use the `_since` param in a cache-friendly manner?  If all clients did something naive like use the timestamp they last polled, then it seems like each request would be a cache-miss.